presto-main/src/test/java/com/facebook/presto/cost/BaseStatsCalculatorTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cost;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class BaseStatsCalculatorTest
+{
+    private StatsCalculatorTester tester;
+
+    @BeforeMethod
+    public void setUp()
+    {
+        tester = new StatsCalculatorTester();
+    }
+
+    @AfterMethod
+    public void tearDown()
+    {
+        tester.close();
+        tester = null;
+    }
+
+    protected StatsCalculatorTester tester()
+    {
+        return tester;
+    }
```

Comment:
you could even go one step further (in future :)) and expose methods like `assertStats`. That way it will be not needed to call `tester().assertStats()`